src/formatDate.js

```diff
@@ -8,7 +8,31 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
```

Comment:
The function 'formatDate' lacks error handling for unexpected scenarios. Although the task specifies that the input will always be correctly formatted, in a real-world scenario, it would be prudent to handle potential mismatches or errors in the input format.